### DISCUSSION TITLE
Chiusura delle nomine degli Ambassador in data 30 Settembre

### DISCUSSION THREAD

**Discussion by Cadienvan** [Il-Libro-Open-Source/book#59] [Annunci]
Abbiamo superato i 20 ambassador e tanti altri si sono candidati e si continuano a candidare, ma d'accordo con l'attuale *formazione*, in data 13 Settembre (Trovate [qui](https://github.com/Il-Libro-Open-Source/book/blob/main/notes/meeting-13-09-2023.md) le note) si è deciso che Sabato 30 Settembre sarà l'ultimo giorno valido per candidarsi per il ruolo, con lo stesso giorno a chiudere la possibilità di votazione.

Dopo tale data ci si potrà comunque proporre, ma non abbiamo ancora stabilito una data di ri-apertura dei sondaggi, presumibilmente tra un trimestre all'incirca, dando il tempo all'attuale formazione di stabilizzarsi, definire regole e buttare giù una prima bozza di contenuto.

Ringraziamo anticipatamente tutti coloro che stanno supportando questo progetto e la squadra di Ambassador, siete tantissimi e non me lo aspettavo assolutamente.

**Comment by guizzo** [comment]
Chiudo in quanto passati oltre 30 giorni dalla data di chiusura delle nomine
